File: icd0017-2018s/radio/app/src/main/java/com/example/radio2019/Domain/Station.java
package com.example.radio2019.Domain;

public class Station {
    public int id;
    public String stationName;

    // ---- meta ----
    public String stationListenUrl;
    public String stationJSONUrl;

    public Station(String stationName, String stationListenUrl, String stationJSONUrl){
        this(0, stationName, stationListenUrl, stationJSONUrl);
    }

    public Station(int id, String stationName, String stationListenUrl, String stationJSONUrl){
        this.id = id;
        this.stationName = stationName;
        this.stationListenUrl = stationListenUrl;
        this.stationJSONUrl = stationJSONUrl;
    }
}

File: icd0017-2018s/Radio_iOS/Radio_iOS/Domain/Station.swift
//
//  Station.swift
//  Radio_iOS
//
//  Created by user154645 on 5/11/19.
//  Copyright © 2019 user154645. All rights reserved.
//

import Foundation

class Station {
    var id: Int
    var stationName: String
    
    var stationListenUrl: String
    var stationJSONUrl: String
    
    convenience init(stationName: String, stationListenUrl: String, stationJSONUrl: String){
        self.init(id: 0, stationName: stationName, stationListenUrl: stationListenUrl, stationJSONUrl: stationJSONUrl)
    }
    
    init(id: Int, stationName: String, stationListenUrl: String, stationJSONUrl: String){
        self.id = id
        self.stationName = stationName
        self.stationListenUrl = stationListenUrl
        self.stationJSONUrl = stationJSONUrl
    }
}
